README.md

```diff
@@ -1,163 +1,183 @@
+
+# Related Publications #
+Please cite this work if you make use of our system in any of your own endeavors:
+
+* **[ElasticFusion: Dense SLAM Without A Pose Graph](http://thomaswhelan.ie/Whelan15rss.pdf)**, *T. Whelan, S. Leutenegger, R. F. Salas-Moreno, B. Glocker and A. J. Davison*, RSS '15
+
+# 1. What do I need to build it? #
+
+## 1.1. Ubuntu ##
+
+* Ubuntu 14.04, 15.04 or 16.04 (Though many other linux distros will work fine)
+* CMake
+* OpenGL
+* [CUDA >= 7.0](https://developer.nvidia.com/cuda-downloads)
+* [OpenNI2](https://github.com/occipital/OpenNI2)
+* SuiteSparse
+* Eigen
+* zlib
+* libjpeg
+* [Pangolin](https://github.com/stevenlovegrove/Pangolin)
+
+Firstly, add [nVidia's official CUDA repository](https://developer.nvidia.com/cuda-downloads) to your apt sources, then run the following command to pull in most dependencies from the official repos:
+
+```bash
+sudo apt-get install -y cmake-qt-gui git build-essential libusb-1.0-0-dev libudev-dev openjdk-7-jdk freeglut3-dev libglew-dev cuda-7-5 libsuitesparse-dev libeigen3-dev zlib1g-dev libjpeg-dev
+```
+
+Afterwards install [OpenNI2](https://github.com/occipital/OpenNI2) and [Pangolin](https://github.com/stevenlovegrove/Pangolin) from source. Note, you may need to manually tell CMake where OpenNI2 is since Occipital's fork does not have an install option. It is important to build Pangolin last so that it can find some of the libraries it has optional dependencies on. 
+
+When you have all of the dependencies installed, build the Core followed by the GUI. 
+
+## 1.2. Windows - Visual Studio ##
+* Windows 7/10 with Visual Studio 2013 Update 5 (Though other configurations may work)
+* [CMake] (https://cmake.org/)
+* OpenGL
+* [CUDA >= 7.0](https://developer.nvidia.com/cuda-downloads)
+* [OpenNI2](https://github.com/occipital/OpenNI2)
+* [SuiteSparse] (https://github.com/jlblancoc/suitesparse-metis-for-windows)
+* [Eigen] (http://eigen.tuxfamily.org)
+* [Pangolin](https://github.com/stevenlovegrove/Pangolin)
+  * zlib (Pangolin can automatically download and build this)
+  * libjpeg (Pangolin can automatically download and build this)
+
+Firstly install cmake and cuda. Then download and build from source OpenNI2, SuiteSparse. Next download Eigen (no need to build it since it is a header-only library). Then download and build from source Pangolin but pay attention to the following cmake settings. There will be a lot of dependencies where path was not found. That is OK except OPENNI2 and EIGEN3 (those should be set to valid paths). You also need to set MSVC_USE_STATIC_CRT to false in order to correctly link to ElasticFusion projects. Also, you can set BUILD_EXAMPLES to false since we don't need them and some were crashing on my machine.
+
+Next, build Core from source. Note that the build might crash if you are missing dirent.h (If so, look here https://github.com/tronkko/dirent). Finally, build GUI.
```

Comment:
Sorry for this. It seems that dirent.h could have been simply omitted on Windows. I have commited the change now.